index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <!-- font awesome -->
+    <script src="https://kit.fontawesome.com/833e2db6e5.js" crossorigin="anonymous"></script>
+
+    <!-- Google Font Montserrat Prompt -->
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:300Prompt&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="main.css">
+    <title>Html Forms</title>
+</head>
+
+<!-- cp am 1 -->
+

```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history